content/plugins/file.md

```diff
@@ -47,7 +47,7 @@ file DBFILE [ZONES... ] {
 Load the `example.org` zone from `example.org.signed` and allow transfers to the internet, but send
 notifies to 10.240.1.1
 
-~~~ corefile
```

Comment:
these are used to do some automated testing, I believe. is this change necessary?

Reply:
That one is auto synced and therefore a merged thing within coredns/coredns.

Reply:
Should be fixed with coredns/coredns#3051 and the next automatic run with version >1.5.2